crates/rpc/src/dto/class.rs

```diff
@@ -9,40 +9,11 @@ impl SerializeForVersion for types::CairoContractClass {
         &self,
         serializer: crate::dto::Serializer,
     ) -> Result<crate::dto::Ok, crate::dto::Error> {
-        impl SerializeForVersion for types::ContractEntryPoints {
-            fn serialize(
-                &self,
-                serializer: crate::dto::Serializer,
-            ) -> Result<crate::dto::Ok, crate::dto::Error> {
-                let mut serializer = serializer.serialize_struct()?;
-
-                serializer.serialize_iter(
-                    "CONSTRUCTOR",
-                    self.constructor.len(),
-                    &mut self.constructor.iter(),
-                )?;
-
-                serializer.serialize_iter(
-                    "EXTERNAL",
-                    self.external.len(),
-                    &mut self.external.iter(),
-                )?;
-
-                serializer.serialize_iter(
-                    "L1_HANDLER",
-                    self.l1_handler.len(),
-                    &mut self.l1_handler.iter(),
-                )?;
-
-                serializer.end()
-            }
-        }
-
         let mut serializer = serializer.serialize_struct()?;
 
         serializer.serialize_field("program", &self.program)?;
         serializer.serialize_field("entry_points_by_type", &self.entry_points_by_type)?;
-        serializer.serialize_optional("abi", self.abi)?;
+        serializer.serialize_optional_with_null("abi", self.abi.clone())?;
```

Comment:
Why the change? I suppose the ABI is rarely missing, so it doesn't matter much in practice, but in case it is, this changes the existing behavior - right?

Reply:
You're right, the ABI is rarely missing, but we do have one test (`types::request::tests::serde::broadcasted_transaction`) with a `CairoContractClass` which is serialized using `serde` (because we still need `serde` for classes) with an `abi: None`, which `serde` serializes as `null`.

So yeah, this was mostly done to satisfy our current test suite. Should not hurt regardless. Let me know if you think otherwise :)

Reply:
Upon reflection, I think the serialization to null was actually implemented by some serde attribute on the abi field - but of course I can't find it now... :-)